examples/subplots_axes_and_figures/align_labels_demo.py

```diff
@@ -4,10 +4,10 @@
 ===============
 
 Aligning xlabel and ylabel using
-:func:`~matplotlib.figure.align_xlabels` and
-:func:`~matplotlib.figure.align_xlabels`
+:func:`matplotlib.figure.align_xlabels` and
```

Comment:
for future reference, you don't need the `:func:` (or `:meth:` or `:class:`, etc.) here: the default role resolves any python object.